lua/plenary/collections/py_list.lua

```diff
@@ -128,20 +130,24 @@ function List:reverse()
   return self
 end
 
+local function iter_forward(seq, i)
```

Comment:
I would prefer they be called `forward_list_gen` or `backward_list_gen` just for consistency with the main library and use `param` and `state` as arguments.

Reply:
gotcha